examples/default.yaml

```diff
@@ -166,6 +166,11 @@ caCerts:
   #   YOUR-ORGS-TRUSTED-CA-CERT-HERE
   #   -----END CERTIFICATE-----
 
+# Upgrade the instance on first boot
```

Comment:
Are you sure it only happens on first boot? We intentionally change the instance id on each boot to make sure cloud-init executes the first-boot logic on each reboot (e.g. to update network configurations). See #273.

Reply:
I don't think there would be too many packages between the reboots, anyway ? But that's what the docs said:

https://cloudinit.readthedocs.io/en/stable/reference/examples.html#update-apt-database-on-first-boot